src/main/java/com/datapipe/jenkins/vault/credentials/VaultCredential.java

```diff
@@ -21,4 +21,22 @@ public String getName(@NonNull VaultCredential credentials) {
             return credentials.getDescription();
         }
     }
+
+    final class VaultAuthorizationResult {
```

Comment:
This class was needed to return the actual token from authorizeWithVault instead of requiring another call to Vault to lookup the token after the fact (since it is unretrievable from a configured Vault class instance).